tools/mppncscatter/mppncscatter.c

```diff
@@ -1415,11 +1417,14 @@ int mppncscatter(mnsopts* opts)
     fflush(stdout);
   }
 
-  nc_close(nc);
+  CHECK_NC_ERRSTAT(nc_sync(nc));
+  CHECK_NC_ERRSTAT(nc_close(nc));
```

Comment:
I can't find whether this is a dryrun or files have been updated and the sync is needed prior to closing.

Reply:
I removed this one, `nc` gets set on line 1289 and its opened with `NC_NOWRITE`. The loop below just needed the sync.